pkg/controller/kogitoinfra/kogitoinfra_controller.go

```diff
@@ -218,18 +218,19 @@ func (r *ReconcileKogitoInfra) Reconcile(request reconcile.Request) (result reco
 }
 
 // updateBaseStatus updates the base status for the KogitoInfra instance
-func (r *ReconcileKogitoInfra) updateBaseStatus(instance *appv1alpha1.KogitoInfra, result *reconcile.Result, err *error) {
+func (r *ReconcileKogitoInfra) updateBaseStatus(instance *appv1alpha1.KogitoInfra, err *error) {
 	log.Info("Updating Kogito Infra status")
-	result = &reconcile.Result{}
```

Comment:
What was the original reason to set this result to empty?

Reply:
To have a reference of the `Result` handled by the `defer`. In other controllers this makes sense, but not in this one. Copy & Paste leftover.